Задания с 18.03.2022/Task1/Task1/Graph.cs

```diff
@@ -0,0 +1,126 @@
+﻿namespace Routers;
+
+using System.Collections.Generic;
+
+/// <summary>
+/// Класс графа
+/// </summary>
+public class Graph
+{
+    private Node[] nodes;
+    private int count = 0;
+    private const int MAX_SIZE = 1000;
+
+    public Graph()
+    {
+        nodes = new Node[MAX_SIZE];
+    }
+
+    public Graph(Node[] nodes)
+    {
+        this.nodes = nodes;
+        count = nodes.Length;
+    }
+
+    /// <summary>
+    /// Получить все вершины графа
+    /// </summary>
+    public Node[] Nodes { get { return nodes; } }
```

Comment:
```suggestion
    public Node[] Nodes => nodes;
```
Это буквально то же самое. А вообще, можно было эти свойства автоматическими сделать